apis/dataplane/v1beta1/common.go

```diff
@@ -102,10 +102,6 @@ type AnsibleOpts struct {
 
 // NodeSection defines the top level attributes inherited by nodes in the CR.
 type NodeSection struct {
-	// ExtraMounts containing files which can be mounted into an Ansible Execution Pod
-	// +kubebuilder:validation:Optional
-	ExtraMounts []storage.VolMounts `json:"extraMounts,omitempty"`
```

Comment:
this is changing the API? isn't it a breaking API change?

Reply:
currently, our code only uses the `ExtraMounts` from `NodeTemplate`: https://github.com/openstack-k8s-operators/openstack-operator/blob/main/apis/dataplane/v1beta1/openstackdataplanenodeset_types.go#L199
We don't have any mechanism to custom the `ExtraMounts` per node, that's why I'm dropping it from the `NodeSection`

Reply:
ack, thanks for the explanation.